samples/bluetooth/direct_test_mode/boards/nrf5340dk_nrf5340_cpunet.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_SPI=n
```

Comment:
This seems to be redundant, as for `SOC_NRF5340_CPUNET` `SPI` [should not be enabled](https://github.com/nrfconnect/sdk-nrf/pull/3927/files#diff-257b50903bc88306185b381540344de79e2c1c201bd7eced9faadeda44862328R13).

Reply:
Right. It seems like I can remove it. Done